tests/visualization_tests/matplotlib_tests/test_param_importances.py

```diff
@@ -25,24 +28,56 @@ def test_plot_param_importances() -> None:
 
     # Test with a trial.
     figure = plot_param_importances(study)
+
+    bars = figure.findobj(Rectangle)[:-1]  # the last Rectangle is the plot itself
+    plotted_data = [bar.get_width() for bar in bars]
+    # get_yticklabels return a data structure of Text(0, 0, 'param_d')
+    labels = [label.get_text() for label in figure.get_yticklabels()]
+
     assert len(figure.get_lines()) == 0
+    assert len(bars) == 2
+    assert set(labels) == set(("param_b", "param_d"))  # "param_a", "param_c" are conditional.
+    assert math.isclose(1.0, sum(i for i in plotted_data), abs_tol=1e-5)
     assert figure.xaxis.label.get_text() == "Importance for Objective Value"
 
     # Test with an evaluator.
     plot_param_importances(study, evaluator=MeanDecreaseImpurityImportanceEvaluator())
+
+    bars = figure.findobj(Rectangle)[:-1]  # the last Rectangle is the plot itself
+    plotted_data = [bar.get_width() for bar in bars]
+    labels = [label.get_text() for label in figure.get_yticklabels()]
+
     assert len(figure.get_lines()) == 0
+    assert len(bars) == 2
+    assert set(labels) == set(("param_b", "param_d"))  # "param_a", "param_c" are conditional.
+    assert math.isclose(1.0, sum(i for i in plotted_data), abs_tol=1e-5)
     assert figure.xaxis.label.get_text() == "Importance for Objective Value"
 
     # Test with a trial to select parameter.
     figure = plot_param_importances(study, params=["param_b"])
+
+    bars = figure.findobj(Rectangle)[:-1]  # the last Rectangle is the plot itself
+    plotted_data = [bar.get_width() for bar in bars]
+    labels = [label.get_text() for label in figure.get_yticklabels()]
+
     assert len(figure.get_lines()) == 0
+    assert len(bars) == 1
+    assert set(labels) == set(("param_b",))  # "param_a", "param_c" are conditional.
```

Comment:
This is really a nitpick. You can align it even further with Plotly's test with the following. Perhaps because `"param_d"` is also omitted (without being addressed in this comment) here because of the explicit `params` argument.
```suggestion
    assert set(labels) == set(("param_b",))
```